website/docs/configuration/functions/sum.html.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: "functions"
+page_title: "sum - Functions - Configuration Language"
+sidebar_current: "docs-funcs-collection-sum"
+description: |-
+  The sum function takes a list of numbers and returns the sum of those
+  numbers.
+---
+
+# `sum` Function
+
+-> **Note:** This page is about Terraform 0.12 and later. For Terraform 0.11 and
+earlier, see
+[0.11 Configuration Language: Interpolation Syntax](../../configuration-0-11/interpolation.html).
```

Comment:
This PR should be merged onto master, and first released in 0.13, so it would be helpful to note that here somehow.

Reply:
Thanks @alisdair, I will make those updates ASAP.

I need to reauthor these commits to be from my work email address, and hence resign the HashiCorp Contributor License Agreement using that address. When I rewrite the author for these commits locally, will doing a `--force push` affect this PR in any negative way? Such as close it out?

Reply:
Thanks!

Force pushing shouldn't cause any problems. I'm not sure the CLA bot will notice the change, though.

If it doesn't, hopefully closing this PR and opening a new one (with a link back here) will do the trick. If you end up doing that, please assign the PR to me so that I don't miss it!

Reply:
Awesome will do. Thanks!!

Reply:
So it rewrote the whole commit history, making it difficult to tell what files actually changed. Is it easier for me to close out and create a new merge request from a fresh clone?

Reply:
Huh. I'm not sure what happened there. Yeah, I think a new PR might be the easiest approach.